charts/ocis/values.yaml

```diff
@@ -239,134 +541,251 @@ features:
-            # -- Ingress class to use.
-            # Uses the default ingress class if not set.
-            ingressClassName:
-            # -- Ingress annotations.
-            annotations: {}
-            # -- Labels for the ingress.
-            labels: {}
-            # -- Ingress TLS configuration.
-            tls: []
-            #  - secretName: chart-example-tls
-            #    hosts:
-            #      - collabora-wopi.owncloud.test
-
-        - # -- Name of the office suite. Will be displayed to the users.
-          name: OnlyOffice
-          # -- Product type of the office suite. Will be used to determine certain product specifics.
-          # Supports "Collabora", "OnlyOffice", "Microsoft365" or "MicrosoftOfficeOnline".
-          product: OnlyOffice
-          # -- Enables the office suite.
-          enabled: false
-          # -- URI of the office suite.
-          uri: ""
-          # -- Description of the office suite
-          description: "Open office documents with OnlyOffice"
-          # -- URI for the icon of the office suite. Will be displayed to the users.
-          iconURI: ""
-          # -- Disables SSL certificate checking for connections to the office suites http api.
-          # Not recommended for production installations.
-          insecure: false
-          # -- Disables verifying requests via WOPI proof keys.
-          # Not recommended to be disabled for production installations.
-          disableProof: false
-          # -- Enable secure view for this office suite. Note: OnlyOffice doesn't support secureView right now
-          secureViewEnabled: false
-          # -- Disables the chat in the office suite. Note: This currently only applies to OnlyOffice
-          disableChat: false
-          # Ingress for collaboration service.
-          ingress:
-            # -- Enables the Ingress. Only needed if the office application is not running within the same cluster.
-            enabled: false
-            # -- Domain of the Ingress.
-            domain: ""
-            # -- Ingress class to use.
-            # Uses the default ingress class if not set.
-            ingressClassName:
-            # -- Ingress annotations.
-            annotations: {}
-            # -- Labels for the ingress.
-            labels: {}
-            # -- Ingress TLS configuration.
-            tls: []
-            #  - secretName: chart-example-tls
-            #    hosts:
-            #      - onlyoffice-wopi.owncloud.test
+      # @schema
+      # type: array
+      # required: true
+      # items:
+      #   type: object
+      #   additionalProperties: false
+      #   properties:
+      #     name:
+      #       type: string
+      #       required: true
+      #     product:
+      #       enum: [Collabora, OnlyOffice, Microsoft365, MicrosoftOfficeOnline]
+      #       required: true
+      #     enabled:
+      #       type: boolean
+      #       required: true
+      #     uri:
+      #       type: string
```

Comment:
We could use https://json-schema.org/understanding-json-schema/reference/string#resource-identifiers here.

Reply:
done in https://github.com/owncloud/ocis-charts/pull/811/commits/48be379079cd7e2f4568298352732dc3baf97c53